FIAT/check_format_variant.py

```diff
@@ -27,6 +27,8 @@
 
 
 def check_format_variant(variant, degree):
+    splitting, variant = parse_lagrange_variant(variant, integral=True)
```

Comment:
Is this to enable macro kinds of variants in non-Lagrange (e.g. BDM, Nedelec) elements?